cpp/pmft/PMFTXYZ.cc

```diff
@@ -70,28 +70,25 @@ void PMFTXYZ::reduce()
  */
 void PMFTXYZ::accumulate(const locality::NeighborQuery* neighbor_query, quat<float>* query_orientations,
                          vec3<float>* query_points, unsigned int n_query_points,
-                         quat<float>* face_orientations, unsigned int n_faces,
+                         quat<float>* equiv_orientations, unsigned int n_faces,
                          const locality::NeighborList* nlist, freud::locality::QueryArgs qargs)
 {
     // precalc some values for faster computation within the loop
     neighbor_query->getBox().enforce3D();
     accumulateGeneral(neighbor_query, query_points, n_query_points, nlist, qargs,
                       [=](const freud::locality::NeighborBond& neighbor_bond) {
                           // create the reference point quaternion
-                          quat<float> ref_q(query_orientations[neighbor_bond.point_idx]);
+                          quat<float> ref_q(query_orientations[neighbor_bond.query_point_idx]);
                           // make sure that the particles are wrapped into the box
                           vec3<float> delta(bondVector(neighbor_bond, neighbor_query, query_points));
 
                           for (unsigned int k = 0; k < n_faces; k++)
```

Comment:
Need to rename `n_faces` to something else like `n_equiv_orientations` -- match the rest of the codebase.